web/ui/src/Services/view-apps.html

```diff
@@ -39,7 +39,8 @@ <h3 class="pull-left" translate>applications</h3>
           <td data-title="'vhost_names'|translate">
               <span ng-if="!service.deploying && !service.isIsvc()">
                   <span ng-repeat="vhost in aggregateVHosts(service) track by $index">
-                      <a href="{{createVHostURL(vhost)}}" target="_blank" class="link">{{createVHostURL(vhost)}}</a>
+                      <health-status ng-if="service.model.DesiredState !== 1 || !vhost.Enabled">{{createVHostURL(vhost)}}</health-status>
```

Comment:
The `isServiceRunning` method on `ServicesController` checks against `service.desiredState` where this one checks against `service.model.desiredState`. These 2 values may not always be indentical, so i recommend choosing one as the source of truth (probably `service.desiredState` since it is most relevant to the UI).